lightning_pose/data/datasets.py

```diff
@@ -119,33 +101,32 @@ def __init__(
         self.keypoints = torch.tensor(csv_data.to_numpy(), dtype=torch.float32)
         # convert to x,y coordinates
         self.keypoints = self.keypoints.reshape(self.keypoints.shape[0], -1, 2)
-        if pytorch_transform_list is None:
-            pytorch_transform_list = []  # make the None an empty list
-        pytorch_transform_list += [
+
+        # send image to tensor and normalize
+        pytorch_transform_list = [
             transforms.ToTensor(),
             transforms.Normalize(mean=_IMAGENET_MEAN, std=_IMAGENET_STD),
         ]
-
         self.pytorch_transform = transforms.Compose(pytorch_transform_list)
+
         # keypoints has been already transformed above
         self.num_targets = self.keypoints.shape[1] * 2
         self.num_keypoints = self.keypoints.shape[1]
 
     @property
-    def height(self):
+    def height(self) -> int:
         # assume resizing transformation is the last imgaug one
         return self.imgaug_transform[-1].get_parameters()[0][0].value
 
     @property
-    def width(self):
+    def width(self) -> int:
         # assume resizing transformation is the last imgaug one
         return self.imgaug_transform[-1].get_parameters()[0][1].value
 
     def __len__(self) -> int:
         return len(self.image_names)
 
-    @typechecked
-    def __getitem__(self, idx: int):
+    def __getitem__(self, idx: int) -> BaseLabeledExampleDict:
```

Comment:
I'm wondering if at this stage, where the context model is quite established in the code, we can unify the two methods. instead of a big if else, both methods with have a list of images and keypoints. just in context=false this list will be of length 1, and not 5. 
might make the base code slightly longer, but at least it's all in one controlled chunk.

Reply:
I wondered this too - could be possible, let's maybe discuss after this round of edits